### ISSUE TITLE
Stream/Screen Share Doesn't Load When Clicking "Pop-Out".

### ISSUE THREAD

**Issue by Benjamoose** [rauenzi/BBDInstaller#10]
I joined the BetterDiscord server first, and several other users are reporting the same issue;

When clicking "Watch Stream" the stream works fine when embedded directly in the main Discord client window, however, if you click the small "pop-out" button to separate the stream into a separate window, the stream audio continues to play, but the video sticks on the Discord loading animation loop. Making the feature unusable.

**Comment by laendle** [comment]
This repo is solely for the BBDInstaller and not for any issue inside the application. The issue here relates to the application and not the installer, this is why there was no response.

**Comment by Benjamoose** [comment]
Ah, so the installer's repo is completely independent from the actual BetterBandagedDiscord project?

If so, is there another repo in which I can report this?

It's definitely isolated to BetterDiscord vs. the stock Discord app, so it'd be worth reporting to the right place.

**Comment by laendle** [comment]
> it'd be worth reporting to the right place.

Agreed. 

As explained, "**the right place**" is **not here**. 

This place is **only** for issues with BBDInstaller.

**Comment by Benjamoose** [comment]
Yeah, I figured...

I was asking in that last reply if you actually _knew_ the appropriate place if not here, since this is the first place I found regarding a BetterDiscord repo, and the original stuff is also no longer active.

I'm basically asking where the active repo is so that I can forward it to the appropriate place and close this.

Thanks again.


**Edit:** I believe I've found it. For anyone finding this via Google, as I did:
https://github.com/rauenzi/BetterDiscordApp/
